### DISCUSSION TITLE
Autocomplete option to reset value on click

### DISCUSSION THREAD

**Discussion by mweghorst** [formkit/formkit#658] [Ideas]
We would like some of our autocomplete inputs to behave more like a dropdown. Searching and selecting values would not change, but the value would be reset when clicked on the input. This can already be achieved using the `#selection` slot, but it would be amazing if this could be done with a simple prop.

https://formkit.link/669049c8e26327d4a5a689c6bf937bdc

**Comment by sashamilenkovic** [comment]
@mweghorst That would be a cool feature add. Do you have a suggestion for a prop name? Maybe.. removeOnClick? clearOnClick? maybe goes without saying, this would be a prop just used for the single-select autocomplete.

**Reply by mweghorst** [reply]
clearOnClick sounds good! And for the single-select only indeed
